core/simplerouter.go

```diff
@@ -35,7 +45,13 @@ func (router *SimpleRouter) Configure(conf PluginConfigReader) error {
 	router.Log = conf.GetLogScope()
 	router.Timeout = nil
 	router.streamID = conf.GetStreamID("Stream", GetStreamID(conf.GetID()))
-	router.modulators = conf.GetModulatorArray("Modulators", router.Log, ModulatorArray{})
+
+	// init router.modulators with filters only
+	filters := conf.GetFilterArray("Filters", router.Log, FilterArray{})
+	for _, filter := range filters {
+		filterModulator := NewFilterModulator(filter)
```

Comment:
If i use the FilterArray i need to implement the Modulate() method again which i want to avoid.  I think it's easier here to handle only the logic in the router to use filters only. This is also more flexible in case we get other modulator types later on.